### ISSUE TITLE
Exception while adding files from server using .addFile()

### ISSUE THREAD

**Issue by rolljee** [VeliovGroup/Meteor-Files#400]
# Problems facing 
In order to let the user download the file generated by a third party api, i need to use .addFile(), since the process happen server side.

```javascript

let metadata = {
	fileName: path.basename(filepath),
	meta: { idlot: workflowId }
};
let filepath = path.join(prefix, filepath);
Filesupload.addFile(path.normalize(filepath), metadata, (error) => {
	if (error) {
	console.log('Error while linking file to standard collection');
	} else {
	console.log(`${file} successfully linked`);
	}
});
```
## An exception is thrown saying : 

```
I20170410-12:07:06.439(2)? Exception in callback of async function: ReferenceError: fileName is not defined
I20170410-12:07:06.440(2)?     at packages/ostrio_files/files.coffee:1510:58
I20170410-12:07:06.440(2)?     at packages/mongo/collection.js:643:5
I20170410-12:07:06.440(2)?     at args.(anonymous function) (packages/matb33_collection-hooks.js:391:23)
I20170410-12:07:06.440(2)?     at runWithEnvironment (packages/meteor.js:1176:24)
I20170410-12:07:06.441(2)?     at packages/meteor.js:1189:14
I20170410-12:07:06.441(2)?     at packages/mongo/mongo_driver.js:320:7
I20170410-12:07:06.441(2)?     at runWithEnvironment (packages/meteor.js:1176:24)

```
+ File is inserted successfully in the collection and I'm able to download them
+ To me this works well, except for error is thrown, i'm not able to understand why this is still happening even if all the process worked...
+ I think ( because i test many thing ) that the error is comming from your package 
+ Also, can you rewrite the wiki part corresponding to .addFile method since it's not really clear.

Once again, all the process work well, but every time i want to insert a file from third party ( witch often happen, this error is shown in the console ).

**Comment by dr-dimitru** [comment]
Hello @rolljee ,

Looks like duplicate of https://github.com/VeliovGroup/Meteor-Files/issues/392
I'll catch up on this during this week.
Sorry for delay.

**Comment by rolljee** [comment]
I read them all, but none of them seems to answer  correctly on this.
Don't worry, since everything work, even if the exception is thrown, it's not urgent.

**Comment by rolljee** [comment]
Any updates ?

**Comment by dr-dimitru** [comment]
@rolljee published as [v1.7.14](https://github.com/VeliovGroup/Meteor-Files/releases/tag/1.7.14), pardon for delay with it, and thank you for contribution.

Please, support this project by:
 - Star on [GitHub](https://github.com/VeliovGroup/Meteor-Files)
 - Star on [Atmosphere](https://atmospherejs.com/ostrio/files)
